spec/support/page_objects/edit_user.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module PageObjects
+  class EditUser
+    include Capybara::DSL
+    include Rails.application.routes.url_helpers
+
+    def initialize
+      visit edit_user_registration_path
```

Comment:
I'm not sure I like the idea of having the initialize trigger a visit 🤔 ...

Reply:
because then `sign_in_page.submit(email, password)` will behave in different ways depending on if `sign_in_page` was called before or not

Reply:
🤔 I understand your point. I'll add all the `visit` calls inside `submit` method 👍

Reply:
I think it'd be better to have it be a separate call on its own